test/Microsoft.AspNet.Mvc.Formatters.Json.Test/JsonInputFormatterTest.cs

```diff
@@ -38,7 +38,11 @@ public void CanRead_ReturnsTrueForAnySupportedContentType(string requestContentT
             var contentBytes = Encoding.UTF8.GetBytes("content");
 
             var httpContext = GetHttpContext(contentBytes, contentType: requestContentType);
-            var formatterContext = new InputFormatterContext(httpContext, new ModelStateDictionary(), typeof(string));
```

Comment:
Is there adequate coverage of the cases around `ModelName + Path` that crop up? I can't tell from the diff